core/cli/src/lib.rs

```diff
@@ -53,23 +55,27 @@ impl AxonCli {
     pub fn init(axon_version: Version, cli_version: &'static str) -> Self {
         let matches = Command::new("axon")
             .version(cli_version)
-            .arg(
-                Arg::new("config_path")
-                    .short('c')
-                    .long("config")
-                    .help("Axon config path")
-                    .required(true)
-                    .num_args(1),
-            )
-            .arg(
-                Arg::new("genesis_path")
-                    .short('g')
-                    .long("genesis")
-                    .help("Axon genesis path")
-                    .required(true)
-                    .num_args(1),
-            )
-            .subcommand(Command::new("run").about("Run axon process"));
+            .subcommand_required(true)
+            .subcommand(
+                Command::new("run")
+                    .about("Run axon process")
+                    .arg(
+                        Arg::new("config_path")
+                            .short('c')
+                            .long("config")
+                            .help("Axon config path")
+                            .required(true)
+                            .num_args(1),
+                    )
+                    .arg(
+                        Arg::new("genesis_path")
+                            .short('g')
+                            .long("genesis")
+                            .help("Axon genesis path")
+                            .required(true)
+                            .num_args(1),
+                    ),
+            );
```

Comment:
It's just my opinion, but I think it's a good time to switch to [derived commands](https://docs.rs/clap/latest/clap/_derive/index.html) to reduce boilerplate and to make getting arguments less error-prone.

Reply:
Sure.